src/doc/rustc/src/platform-support/armv5te-none-eabi.md

```diff
@@ -0,0 +1,75 @@
+# `armv5te-none-eabi`
+
+**Tier: 3**
+
+Bare-metal target for any cpu in the ARMv5TE architecture family, supporting
+ARM/Thumb code interworking (aka `a32`/`t32`), with ARM code as the default code
+generation.
+
+The `thumbv5te-none-eabi` target is the same as this one, but with THUMB code as the default.
```

Comment:
Rust supports switching the “mode” via the (currently unstable) `instruction_set` attribute, so I would argue that describing this in terms of that attribute would make the intent here clearer:

```suggestion
The `thumbv5te-none-eabi` target is the same as this one, but the instruction set defaults to `t32` instead of `a32`.
```